src/app/[user-name]/layout.tsx

```diff
@@ -9,17 +8,17 @@ export const metadata: Metadata = {
 
 export default async function RootLayout({
   children,
-  params,
-}: {
+} // params,
+: {
   children: React.ReactNode;
   params: { 'user-name'?: string };
 }) {
-  const userId = params?.['user-name'];
-  const user = await getUserDetails(userId);
+  // const userId = params?.['user-name'];
+  // const user = await getUserDetails(userId);
```

Comment:
Is this comment important?
If so, you can add a `TODO: remove after development`

Reply:
ahh no actually that will be added when backend were ready

Reply:
Okay. So do you want to keep it or take it out?